armi/reactor/blocks.py

```diff
@@ -2018,7 +2023,7 @@ def getPinToDuctGap(self, cold=False):
         pinOuterFlatToFlat = (
             pinCenterFlatToFlat
             + clad.getDimension("od", cold=cold)
-            + 2.0 * wire.getDimension("od", cold=cold)
+            + wire.getDimension("od", cold=cold)
```

Comment:
I guess this can go either way, but I thought the factor of 2.0 should remain here. I was considering the `pinToDuctGap < wireThicknesses` as the double-counting. It's algebraically the same as what I was thinking, but just a different way of defining `pinToDuctGap`

Reply:
Yeah you're right. That factor 2 should stay. Bonehead move on my part. 

Can you explain the double-counting thing some more? I'm not quite following.